### PR TITLE
Rails-5 add navigation helper

### PR DESCRIPTION
**Pull request by zoiec**
🐘  🎨 
- Updated member-in-renewal
- Added helper module to be included in apps to determine appropriate navigation
- updated version

Member in Renewal View
image

### CODE REVIEW

**Comment by jessedoyle** [comment]
:elephant::elephant::elephant:

**Comment by sineade** [comment]
@zoiec are there any visual changes? Can we please have screenshots

**Comment by jessedoyle** [comment]
:elephant::elephant::elephant:

**Comment by zoiec** [comment]
👋

**Comment by kaytcampbell** [comment]
:art: :art: :art:

File: lib/ama_layout/navigation_helper.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      end
+    end
+
+    private
+
+    def _member?(member)
+      member
+    end
+
+    def _in_renewal?(in_renewal)
```
Comment (mvandenbeuken) [inline lib/ama_layout/navigation_helper.rb]: @zoiec also in this PR. could these 3 methods just use member?, in_renewal and status == 'AL' without passing member?, in_renewal and status in as args?

**Comment by zoiec** [comment]
👋

**Comment by rubene** [comment]
:book:

File: lib/ama_layout/navigation.yml [hunk 2]
```diff
@@ -81,26 +81,78 @@ non-member:
...
+        link: "<%= Rails.configuration.automotive_site %>/"
+  - text: "My Driver Education"
     link: "<%= Rails.configuration.driveredonline_site %>/"
     icon: "fa-car"
     sub_nav:
       - text: "Driver Education Overview"
         link: "<%= Rails.configuration.driveredonline_site %>/"
       - text: "New Driver Online Program"
         link: "<%= Rails.configuration.driveredonline_site %>/dashboard"
+  - text: "My AMA Rewards"
```
Comment (rubene) [inline lib/ama_layout/navigation.yml]: can you add test for these new links and subnavs here https://github.com/amaabca/ama_layout/blob/e3793d9af45b27ec61a9423a5846c210b6073c96/spec/ama_layout/navigation_spec.rb

**Comment by zoiec** [comment]
👋  @rubene

**Comment by rubene** [comment]
🐘 🐘 🐘

**Pull request merged**